### ISSUE TITLE
"Copy Links" does not work without "Full Disk Access"  (macOS Ventura)

### ISSUE THREAD

**Issue by raoulk** [bettar/miele-lxiv#81]
Hi,

When I import images and choose "Copy Links" instead of "Copy Files", it seems that the import works: I see a new entry in the database, and the preview thumbnails are there. But when I open something in the "2D viewer", there are only white images.

Importing with "Copy Files" works.

I assume it's an issue with the sandbox settings. Probably miele-lxiv has not the permission to access files that are outside of the sandbox.

Other programs ask for permissions to access specific directories, but not miele-lxiv.

Current workaround: I grant miele-lxiv "Full Disk Access" in System Settings ("Privacy & Security"/"Full Disk Access"). But I would prefer to only grant access to the "Documents" folder.

I am still a happy user, thanks for your good work!

**Comment by bettar** [comment]
> I assume it's an issue with the sandbox settings. Probably miele-lxiv has not the permission to access files that are outside of the sandbox.

That's correct. Can anyone recommend where to make such setting ? The current settings are as shown below:
<img width="562" alt="Screenshot 2023-04-11 at 09 37 14" src="https://user-images.githubusercontent.com/9733270/231089737-cb6c1307-c546-439c-ba28-55d09c7e5369.png">
<img width="512" alt="Screenshot 2023-04-11 at 09 37 49" src="https://user-images.githubusercontent.com/9733270/231089856-d4ca3fff-8e4a-4345-833a-c1e7415439b6.png">
